### PR TITLE
EHN: Added lerp in twnsoe math.py

### LINKED ISSUE #19444: lerp

### PR DESCRIPTION
**Pull request by MuhammadNizamani**
Closed #19444

### CODE REVIEW

**Comment by DragosStoican** [comment]
Hi @MuhammadNizamani . Great PR, thank you very much. However, it looks like you've got a few formatting errors [here] (https://github.com/unifyai/ivy/actions/runs/5561769257/jobs/10159654481?pr=19446) . Can you please address those that were caused by your changes and then I'd be happy to merge your PR. Thanks for your contribution!

**Comment by MuhammadNizamani** [comment]
@DragosStoican  review this I have set the  formet

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py [hunk 1]
```diff
@@ -1468,3 +1468,37 @@ def test_paddle_max(
...
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        test_flags=test_flags,
+        on_device=on_device,
+        x=x[0],
+        y=x[1],
+        weight = x[2],
```
Comment (DragosStoican) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:1503]: This is still causing the formatting test to fail
 Remove the spaces around the "="

Comment (MuhammadNizamani) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:1503]: OH!! I did not see that sorry I have set this

**Comment by MuhammadNizamani** [comment]
@DragosStoican  Sir I have set the formetting

**Pull request merged**